code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -737,10 +733,9 @@ nobiliumsuppression = INFINITY
 	var/energy_taken = nob_formed * (NOBLIUM_FORMATION_ENERGY / (max(cached_gases[/datum/gas/bz][MOLES], 1)))
 	if ((cached_gases[/datum/gas/tritium][MOLES] - 5 * nob_formed < 0) || (cached_gases[/datum/gas/nitrogen][MOLES] - 10 * nob_formed < 0))
 		return NO_REACTION
-	cached_gases[/datum/gas/tritium][MOLES] -= nob_formed * 5
-	cached_gases[/datum/gas/nitrogen][MOLES] -= nob_formed * 10
-	cached_gases[/datum/gas/hypernoblium][MOLES] += nob_formed
-	SSresearch.science_tech.add_point_type(TECHWEB_POINT_TYPE_DEFAULT, nob_formed * NOBLIUM_RESEARCH_AMOUNT)
+	cached_gases[/datum/gas/tritium][MOLES] -= 10*nob_formed
+	cached_gases[/datum/gas/nitrogen][MOLES] -= 20*nob_formed
```

Comment:
real minor thing, but spaces by the * please, there is an attempt in this file to make it clean. Key word being attempt

Reply:
im not even sure why the values were changed, I'll revert that